lib/rouge/lexers/praat.rb

```diff
@@ -156,7 +176,7 @@ def self.detect?(text)
 
         rule %r/"/, Literal::String, :string
 
-        rule %r/\b(?:#{objects.join('|')})(?=\s+\S+\n)/, Name::Class, :string_unquoted
```

Comment:
It doesn't actually save all that much memory if you immediately join the class objects like this. Usually it's better to test inclusion within a block action. See for example https://github.com/rouge-ruby/rouge/blob/master/lib/rouge/lexers/haskell.rb#L66

Reply:
Thanks, @jneen. Let me see if we can convert this to use block action.